yt/chyt/server/query_context.h

```diff
@@ -278,3 +281,7 @@ void InvalidateCache(
 ////////////////////////////////////////////////////////////////////////////////
 
 } // namespace NYT::NClickHouseServer
+
+#define QUERY_CONTEXT_INL_H_
+#include "query_context-inl.h"
```

Comment:
There is no `query_context-inl.h` in this PR

Reply:
That's a pity! Added id, so you can import again for tests.